api/docs/db/DataCollections.md

```diff
@@ -0,0 +1,131 @@
+runstatus | str | Popuplate as `Successful` on completion
+imagedirectory | str | The directory where images / data are recorded
+imageprefix | str | The prefix for images i.e. `filename_`, for filename_0001.cbf 
+filetemplate | str | The template for the images / data. For a series of images this would be filename_####.cbf, for a hdf5 file the normal name can be used filename.h5
+imagecontainersubpath | str | A path to the data if its a hdf5 file
+
+For mx oscillation data
+
+Column | Type | Description
+--- | --- | ---
+axisstart | float | Oscillation start in degrees
+axisend | float | Oscillation end in degrees
+axisrange | float | Oscillation range in degrees, 0.1deg
+rotationaxis | str | The roation axis used: omega, phi, ...
+overlap | float | Overlap for screening collection, -44.5 for 0, 45, 90 with oscillation of 0.5deg
+numberofimages | int | Number of images collected
+exposuretime | float | Exposure time in s
+wavelength | float | Wavelength of beamline in angstrom
+detectordistance | float | Detector distance in mm
+resolution | float | Resolution at edge of detector in angstrom
+transmission | float | Transmission in %, i.e. 10
+xtalSnapshotFullPath1 | path | Path to a snapshot of the crystal at 0
+xtalSnapshotFullPath2 | path | Path to a snapshot of the crystal at 90 (for gridscan with grid and results annotated)
+xtalSnapshotFullPath3 | path | Path to a snapshot of the crystal at 180 (for gridscans with grid annotated)
+phistart | float | For multi-axis goniomnetry, phi angle at start
+kappastart | float | For multi-axis goniometry, kappa angle at start
+omegastart | float | For multi-axis goniometry, omega angle at start
+chistart | float | For multi-axis goniometry, chi angle at start
+beamsizeatsamplex | float | Beamsize at the sample in microns in horizontal
+beamsizeatsampley | float | Beamsize at the sample in microns in vertical
+xbeam | float | Position of beam in horizontal
+ybeam | float | Position of beam in vertical
+detectorid | int | A detectorid from `Detector`
+
+
+Multiple data collections can be stored in the same group, generally this applies to things like wedged SAD collections.
+
+
+## Types
+
+Data collections are classified using some fairly cryptic column combinations. These should be consolidated and simplified in the long term.
+
+### Screening
+
+A screening data collection is inferred from `DataCollection.overlap` > 0
+
+### Grid Scans
+
+`DataCollectionGroup.experimentType` should be populated as `Mesh`, although grid is actually inferred from:
+```
+DataCollection.numberOfImages > 1 && DataCollection.axisrange = 0
+```
+In the long term this should be consolidated to use `DataCollectionGroup.experimentType`
+
+For gridscans the third crystal snapshot is used `DataCollection.xtalsnapshotfullpath3`. Interally this is because `xtalsnapshotfullpath1` is of the crystal, `xtalsnapshotfullpath2` has the grid with results annotated, and `xtalsnapshotfullpath3` has the grid alone. 
```

Comment:
```suggestion
For gridscans the third crystal snapshot is used `DataCollection.xtalsnapshotfullpath3`. Internally this is because `xtalsnapshotfullpath1` is of the crystal, `xtalsnapshotfullpath2` has the grid with results annotated, and `xtalsnapshotfullpath3` has the grid alone. 
```